src/main/java/seedu/momentum/ui/ProjectCard.java

```diff
@@ -25,8 +24,7 @@
      * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on ProjectBook level 4</a>
```

Comment:
Small issue but can you just change `ProjectBook` to `addressbook` here because the issue comes from there